ibm/service/kms/resource_ibm_kms_key_policies_test.go

```diff
@@ -40,6 +40,38 @@ func TestAccIBMKMSKeyPolicy_basic_check(t *testing.T) {
 	})
 }
 
+func TestAccIBMKMSKeyPolicy_basic_check_enable(t *testing.T) {
+	instanceName := fmt.Sprintf("kms_%d", acctest.RandIntRange(10, 100))
+	keyName := fmt.Sprintf("key_%d", acctest.RandIntRange(10, 100))
+	rotationEnable := true
+	rotationBool := false
```

Comment:
```suggestion
	rotationDisable := false
```

Reply:
Thanks for the input, I did not changed bbecause rotationDisbale = false was a little misleading.